docs/guides/install/index.html

```diff
@@ -3,215 +3,248 @@
-    <div class="hxAlert">
-      <div class="hxAlert__icon">
-        <hx-icon type="info-circle"></hx-icon>
-      </div>
-      <div class="hxAlert__text">
-        <span class="hxAlert__status">NOTE</span>
-        Currently, the loader only supports being loaded in the <code>&lt;head&gt;</code>
-        of an HTML document.
-      </div>
-    </div>
-    <br />
-
-    <p>
-      The web components loader comes from the
-      <a href="https://github.com/webcomponents/webcomponentsjs">webcomponentsjs</a>
-      project. Its job is to perform feature detection on the current browser
-      so that it can dynamically inject the smallest polyfill needed to
-      support web components into the head of the HTML document.
-    </p>
+  <section><!-- Select an Application Layout -->
+    <h2 class="hxSectionTitle" id="application-layout">Select an Application Layout</h2>
     <p>
-      Once the polyfills are loaded (if needed), the "WebComponentsReady" event
-      will be fired on the document to announce availability of Web Component
-      functionality.
+      You have two choices regarding an application layout.
     </p>
-    <ul>
+    <ol>
       <li>
-        <code>Array.from</code> is polyfilled along with the
-        <code>webcomponents-lite.js</code> polyfill.
+        Start with an official
+        <a href="components/layouts">application layout</a>
+        <small>(recommended)</small>
       </li>
       <li>
-        A <a href="https://github.com/webcomponents/webcomponentsjs/pull/861">pull request</a>
-        is open to allow inserting the loader in the <code>&lt;body&gt;</code>.
+        Continue reading to learn how to create a custom layout.
       </li>
-    </ul>
-    {% code 'html' %}
-      <head>
-        ...
-
-        <script src="cdn/path/to/webcomponents-loader.js"></script>
-        <!-- polyfills injected here (if needed) -->
-      </head>
-    {% endcode %}
+    </ol>
   </section>
 
-  <section><!-- ES5 Adapter -->
-    <h3 class="hxSubSectionTitle" id="es5-adapter">ES5 Adapter</h3>
-    <p>
-      The Custom Elements v1 spec requires that constructor classes be defined
-      using ES6 class syntax.  This is problematic, given that we have to
-      support legacy browsers like Internet Explorer that do not support ES6.
-    </p>
-    <p>
-      The ES5 Adapter corrects this problem by converting ES5 classes to ES6
-      syntax so that modern browsers can load custom element definitions.
-    </p>
-    <p>
-      A complication arises when a legacy browser tries to run the ES5 Adapter.
-      Because the adapter is written in ES6 syntax, it will raise an exception
-      in any browser that doesn't support ES6.  To get around this we can wrap
-      the injection of the ES5 adapter in a container that can be dynamically
-      removed if it isn't needed.
-    </p>
+  <section><!-- Custom Layouts -->
+    <h1 class="hxPageTitle" id="custom-layouts">Custom Layouts</h1>
```

Comment:
Is this how to create a custom layout? If so, I would change this to "Create a custom layout".